README.md

```diff
@@ -284,6 +290,97 @@ $sql = $apiFilter->applyFilters($filters, $sql); // SELECT * FROM person WHERE 1
 $preparedValues = $apiFilter->getPreparedValues($filters, $sql); // ['firstname_eq' => 'John', 'surname_eq' => 'Snow']
 ```
 
+### More Examples
+
+#### Equals (_implicit and explicit_)
+```http request
+GET http://host/person/?fullName=Jon Snow
+GET http://host/person/?fullName[eq]=Jon Snow
+```
+Result:
+```yaml
+-   column: fullName
+    filters: eq
+    value: Jon Snow
+```
+
+#### Multiple filters (_implicit and explicit_)
+By single values
+```http request
+GET http://host/person/?firstName=Jon&surname=Snow
+GET http://host/person/?firstName[eq]=Jon&surname[eq]=Snow
+```
+
+By Tuples
+```http request
+GET http://host/person/?(firstName,surname)=(Jon,Snow)
+GET http://host/person/?(firstName,surname)[eq]=(Jon,Snow)
+GET http://host/person/?(firstName[eq],surname[eq])=(Jon,Snow)
+```
+Result:
+```yaml
+-   column: firstName
+    filters: eq
+    value: Jon
+
+-   column: surname
+    filters: eq
+    value: Snow
+```
+
+#### Multiple filters
+You can mix all types of filters (_tuples, explicit, implicit_).
+
+##### _Perfect wife_ by generic filters
```

Comment:
Still there.

Reply:
and what is wrong with it?

Reply:
Don't you think this could be offensive to someone?

Reply:
So the Tinder should remove their "discovery settings" also? :D